src/cwts/networkanalysis/run/RunNetworkLayout.java

```diff
@@ -0,0 +1,454 @@
+     */
+    public static final int DEFAULT_NORMALIZATION = NO_NORMALIZATION;
+
+    /**
+     * Default attraction parameter.
+     */
+    public static final int DEFAULT_ATTRACTION = GradientDescentVOSLayoutAlgorithm.DEFAULT_ATTRACTION;
+
+    /**
+     * Default repulsion parameter.
+     */
+    public static final int DEFAULT_REPULSION = GradientDescentVOSLayoutAlgorithm.DEFAULT_REPULSION;
+
+    /**
+     * Default number of random starts.
+     */
+    public static final int DEFAULT_N_RANDOM_STARTS = 1;
+
+    /**
+     * Default maximum number of iterations.
+     */
+    public static final int DEFAULT_MAX_N_ITERATIONS = GradientDescentVOSLayoutAlgorithm.DEFAULT_MAX_N_ITERATIONS;
+
+    /**
+     * Default initial step size.
+     */
+    public static final double DEFAULT_INITIAL_STEP_SIZE = GradientDescentVOSLayoutAlgorithm.DEFAULT_INITIAL_STEP_SIZE;
+
+    /**
+     * Default minimum step size.
+     */
+    public static final double DEFAULT_MIN_STEP_SIZE = GradientDescentVOSLayoutAlgorithm.DEFAULT_MIN_STEP_SIZE;
+
+    /**
+     * Default step size reduction.
+     */
+    public static final double DEFAULT_STEP_SIZE_REDUCTION = GradientDescentVOSLayoutAlgorithm.DEFAULT_STEP_SIZE_REDUCTION;
+
+    /**
+     * Default required number of quality value improvements.
+     */
+    public static final int DEFAULT_REQUIRED_N_QUALITY_VALUE_IMPROVEMENTS = GradientDescentVOSLayoutAlgorithm.DEFAULT_REQUIRED_N_QUALITY_VALUE_IMPROVEMENTS;
+
+    /**
+     * Description text.
+     */
+    public static final String DESCRIPTION
+        = "RunNetworkLayout version 1.1.0\n"
+          + "By Nees Jan van Eck and Ludo Waltman\n"
+          + "Centre for Science and Technology Studies (CWTS), Leiden University\n";
+
+    /**
+     * Usage text.
+     */
+    public static final String USAGE
+        = "Usage: RunNetworkLayout [options] <filename>\n"
+          + "\n"
+          + "Determine a layout for a network using the gradient descent VOS layout\n"
+          + "algorithm.\n"
+          + "\n"
+          + "The file in <filename> is expected to contain a tab-separated edge list\n"
+          + "(without a header line). Nodes are represented by zero-index integer numbers.\n"
+          + "Only undirected networks are supported. Each edge should be included only once\n"
+          + "in the file.\n"
+          + "\n"
+          + "Options:\n"
+          + "-n --normalization {" + NORMALIZATION_NAMES[NO_NORMALIZATION] + "|" + NORMALIZATION_NAMES[ASSOCIATION_STRENGTH] + "|" + NORMALIZATION_NAMES[FRACTIONALIZATION] + "|" + NORMALIZATION_NAMES[LINLOG] + "}\n"
+          + "        (Default: " + NORMALIZATION_NAMES[NO_NORMALIZATION] + ")\n"
+          + "    Method for normalizing the edge weights.\n"
+          + "-a --attraction <attraction> (Default: " + DEFAULT_ATTRACTION + ")\n"
+          + "    Attraction parameter of the VOS quality function.\n"
+          + "-r --repulsion <repulsion> (Default: " + DEFAULT_REPULSION + ")\n"
```

Comment:
Perhaps clarify this should be integer valued.

Reply:
See error message.